File: 조준장/level2/카펫.py
def solution(brown, yellow):
    answer = []
    width = int((brown - 4) / 2)
    
    for height in range(1, width+1):
        if (width - height) * height == yellow:
            answer.append(width - height + 2)
            answer.append(height + 2)
            break
            
    return answer

File: 조준장/level2/카펫.kt
class Solution {
    fun solution(brown: Int, yellow: Int): IntArray {
        val width : Int = ((brown - 4) / 2).toInt()
        return (1..yellow+1).first{
            ((width - it) * it == yellow)}.let{
            intArrayOf(width - it + 2 , it + 2)
        }    
    }
}
